server/src/main/resources/applicationContext.xml

```diff
@@ -87,18 +87,26 @@
 
     <!-- Cache Config -->
     <cache:annotation-driven/>
+    
+    <beans profile="ldap,saml,eaz">
```

Comment:
what is eaz profile?

Reply:
we should not update this xml, because this will enable memcache be default for ldap and saml.
we should write these configure on document, users can follow the doc to setup

Reply:
Yes, it's better to put this part on doc.